tools/cpboard.py

```diff
@@ -127,7 +127,7 @@ def reset(self):
         self.write(b"\r" + REPL.CHAR_CTRL_B)  # enter or reset friendly repl
         data = self.read_until(b">>> ")
 
-    def execute(self, code, timeout=10, wait_for_response=False):
+    def execute(self, code, timeout=10, wait_for_response=True):
```

Comment:
This is a very excellent point, and I don't consider it a nit-pick at all.

I don't expect many people are using this. I point to it as an example how to interact with boards via a PC, but that's about it.

A lot of the semantics are so that it remains py2/py3 compatible, from what I understand (its base is MicroPython's `tools/pyboard.py`. I'll likely give it more scrubs for a multitude of reasons. One of those scrubs will include removing Python2 use; not worth using at this point.